docs/sdk/server-side-sdks/dotnet-local/dotnet-local-usage.md

```diff
@@ -258,3 +258,46 @@ namespace Example {
     }
 }
 ```
+
+## Set Client Custom Data
+
+To assist with segmentation and bucketing you can set a custom data dictionary that will be used for all variable and feature evaluations. User specific custom data will override client custom data.
+
+```csharp
+namespace Example {
```

Comment:
If we're doing a fully complete example we should include the import statements.

Reply:
yeah, i don't know why they are different. I was just trying to stay consistent within the page

Reply:
Ok, i updated the dotnet examples to all have import statements to be consistent